### PR TITLE
Literature tool

### PR DESCRIPTION
**Pull request by denvernaaf**
Screen Shot 2020-02-09 at 3 23 19 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +40,7 @@
     "moment-duration-format": "2.2.2",
     "react": "16.5.2",
     "react-bootstrap": "0.32.4",
+    "react-copy-to-clipboard": "^5.0.2",
```
Comment (webservant-mhasc) [inline package.json]: I have used this before!

Comment (denvernaaf) [reply package.json]: It looked pretty good!

File: src/views/LiteratureOrder.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
+import React from 'react'
+import { camelCase } from 'lodash'
```
Comment (webservant-mhasc) [inline src/views/LiteratureOrder.js]: I think you'll want
`import camelCase from 'camelCase/map';`

File: src/views/LiteratureOrder.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+import { formatMoney } from 'accounting'
```
Comment (webservant-mhasc) [inline src/views/LiteratureOrder.js]: Not sure on this package size, you might need to find a way to do the same. If not possible then go to the source and copy the function out of it.

File: src/views/LiteratureOrder.js [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+import withStyles from '@material-ui/core/styles/withStyles'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```
Comment (webservant-mhasc) [inline src/views/LiteratureOrder.js]: same here.

File: src/views/LiteratureOrder.js [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+          </Typography>
+        </Grid>
+        {prices ? (
+          <React.Fragment>
+            {SECTION_NAMES.map(name => (
+              <Grid
+                item
+                xs={12}
+                md={10}
+                // justify="center"
```
Comment (webservant-mhasc) [inline src/views/LiteratureOrder.js]: rm

File: src/views/LiteratureOrder.js [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+          <InputLabel> Quantity</InputLabel>
+          <NativeSelect
+            onChange={(e) => this.onChange(e.target.value, itemNumber, sectionName)}
+            value={quantity}
+            inputProps={{
+              name: 'quantity',
+              id: 'quantity',
+            }}
+          >
+            {ZERO_TO_ONE_HUNDRED.map((number) => (<option value={number} key={number}>{number === 0 ? "none" : number}</option>))}
```
Comment (webservant-mhasc) [inline src/views/LiteratureOrder.js]: We should generate this as a const instead of on every render.

File: src/views/LiteratureOrder.js [hunk 7]
```diff
@@ -0,0 +1,310 @@
...
+      <FormControl >
+        <Select
+          value={quantity}
+          onChange={(e) => this.onChange(e.target.value, itemNumber, sectionName)}
+          inputProps={{
+            name: 'quantity',
+            id: 'quantity',
+          }}
+        >
+          {ZERO_TO_ONE_HUNDRED.map((number) => (<MenuItem value={number} key={number}>{number === 0 ? "none" : number}</MenuItem>))}
```
Comment (webservant-mhasc) [inline src/views/LiteratureOrder.js]: Then reuse the const above here.

File: src/views/LiteratureOrder.js (hunk 6 above)
Comment (webservant-mhasc) [reply src/views/LiteratureOrder.js]: For example, this could be a static thing called zeroToOneHundredOption

**Pull request merged**